stable/consul/README.md

```diff
@@ -32,6 +32,8 @@ The following tables lists the configurable parameters of the consul chart and t
 | `ImagePullPolicy`       | Container pull policy                 | `Always`                                                   |
 | `Replicas`              | k8s statefulset replicas              | `3`                                                        |
 | `Component`             | k8s selector key                      | `consul`                                                   |
+| `ConsulConfig.[].type`  | either `secret` or `configMap`        | -                                                      |
```

Comment:
I don't like the syntax you used. This is confusing. May just add `ConsulConfig` and add an example to `values.yaml`.